deployments/cardano-nodes.nix

```diff
@@ -1,7 +1,9 @@
-{ globals, ... }: with (import ./../lib.nix);
+{ environment, globals, ... }: with (import ./../lib.nix);
 let nodeMap = globals.nodeMap; in
 
-flip mapAttrs nodeMap
+(flip mapAttrs nodeMap
 (name: import ./../modules/cardano.nix
        globals
-       [])
+       [])) // {
+  require = if (environment == "production") && (nodeMap ? r-a-1) then [ ./mainnet-noauto-restart.nix ] else [];
```

Comment:
I realize `nodeMap ? r-a-1` was added to fix a broken test, but as a result `r-a-1` is duplicated.

We may also decide to adjust the set of nodes where cardano-restart is disabled, so it would be nice to support an explicit list, e.g. ``rollingRestartBlacklist = [r-a-1];`, and remove any other mentions of specific hosts for this feature.